backend/protos/mgmt/v1alpha1/transformer.proto

```diff
@@ -515,6 +517,39 @@ message GetTransformPiiEntitiesResponse {
 
 message GenerateBusinessName {}
 
+enum GenerateIpAddressVersion {
+  // Unspecified defaults to ipv4
+  GENERATE_IP_ADDRESS_VERSION_UNSPECIFIED = 0;
+  // Generates a random ipv4
+  GENERATE_IP_ADDRESS_VERSION_V4 = 1;
+  // Generates a random ipv6
+  GENERATE_IP_ADDRESS_VERSION_V6 = 2;
+}
+
+enum GenerateIpAddressClass {
```

Comment:
Could we just collapse the `GenerateIpAddressVersion` into this one? I think most of these only apply to Ipv4 anyways, so it doesn't really make sense to be able to configure v6 along with a Class B address. The way this is defined now leads to invalid combinations.

I would suggest either doing a Oneof where you specify either an Ipv6 Config or an Ipv4 Config, or just have a variety of discrete enums for V4 or V6.
Happy to chat more about this offline if it isn't clear.